HW02_B_Mimic_starter.Rmd

```diff
@@ -1,99 +1,118 @@
-
-
-### Graph 4 
-```{r, echo = T}
-data(mpg)
-
-#hint for the coloring, colorbrewer and you can set palette colors and make your graphs colorblind friendly
-library(RColorBrewer)
-display.brewer.all(colorblindFriendly = T) #take a look at the colorblindfriendly options
-```
-
-The above graph lets you see some colobrlind friendly palettes. For the graph below, I used Set2.
-
-
-
-Now using the above mpg dataset, make this graph
-
-```{r graph 4 code}
-
-```
-
-
-
-
+---
+title: "HW02_B_Graph-Mimic"
+author: "Olivia Pura"
+output: github_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+```
+
+
+```{r load libraries, echo = TRUE, include=TRUE}
+library("ggplot2")
+library("magrittr") #so I can do some piping
+data("diamonds")
+data("mpg")
+data("iris")
+theme_set(theme_bw()) #I'll give you this one, you can set the theme individually for graphs
+#or you can set the theme for all the graphs using theme_set()
+#theme_bw() is best theme (IMO)
+
+#for graph 3:
+#library("ggrepel")
+
+
+```
+
+## HW02 Part B
+
+For this part of the HW, the goal is to try to recreate the graphs I make from scratch. I will only provide the MD, not the actual code I used to create it besides which data I use to create it. The rest will be up to you. 
+
+Try for all 4, but if you are struggling don't worry about it. Try your best for each, if you don't get everything that's what the peer-review is for. :smile:
+
+
+### Graph 1
+```{r, echo = T, include = TRUE}
+
+ggplot(data = diamonds) + 
+  geom_bar(mapping = aes(x = cut, fill = clarity), position = "dodge") +
+  theme(plot.title = element_text(hjust = 0.5)) +
+  labs(x = "Diamond Cut", 
+       y = "Number of Diamonds", 
+       title = "My Diamond Collection", 
+       subtitle = "Barplot representing the number of diamonds in my diamond collection by \ntype of cut quality and clarity of diamond") +
+  annotate("rect", xmin = 4.5, xmax = 5.5, ymin = 0, ymax = 5000,
+           fill = "gray30", alpha  = 0.5,
+           label = "My Best Diamonds,\nof Course") +
+  annotate("text", x = 4, y = 4500, label = "My Best Diamonds,\nof Course") 
+
+```
+
+### Graph 2
+```{r, echo = T, include = TRUE}
+ggplot(data = iris, mapping = aes(x = Sepal.Length, y = Petal.Length)) + 
+  geom_point(mapping = aes(color = Species, shape = Species)) +
+  facet_wrap(~Species, scales = "free_y") +
+  geom_smooth(method = "lm", se = FALSE, color = "black")
+  labs(x = "Sepal.Length", 
+       y = "Petal.Length") 
+```
+
+
+### Graph 3
+You'll need the information in this first box to create the graph
+```{r, echo = TRUE}
+data("mpg")
+corvette <- mpg[mpg$model == "corvette",]
+#install
+require("ggrepel") #useful for making text annotations better, hint hint
+set.seed(42)
+```
+
+Now using the mpg dataset and the corvette dataset, make this graph:
+
+```{r graoh 3 code}
+ggplot() + 
+  geom_point(data = mpg, mapping = aes(x = displ, y = hwy)) +
+  scale_x_continuous(limits = c(1, 8), breaks = seq(1, 9, 1)) +
```

Comment:
I think breaks here are breaks = seq(1, 8, 1), though in this case it returns the same results as seq(1, 9, 1) because of the limits = c(1, 8).